exercises/concept/high-score-board/.meta/exemplar.js

```diff
@@ -0,0 +1,76 @@
+// @ts-check
+
+/**
+ * Creates a new score board with an initial entry.
+ *
+ * @returns {Object.<string, number>} new score board
```

Comment:
@SleeplessByte I am not up to date with this JSDoc ts-check business. :point_up_2:  This is what I found in the JSDoc documentation how to describe the object. Is that ok or should there be something else (more typescripty) here?

Reply:
@junedev you want to represent a shape that has keys of string and values of number?

```typescript
@returns {Record<string, number>}
```

which is essentially the same as:

```typescript
@returns {{ [key: string]: number }}
```

Reply:
> you want to represent a shape that has keys of string and values of number?

Yes. Inserted your suggestion. Thanks!